### PR TITLE
Feat/improve chart tab titles

### PR DESCRIPTION
**Pull request by noahonyejese**
### **This PR is to optimize the UX for adding and editing tab titles**

**Issue**
1. Go to this [Link](https://visualization-tool-5nwbb9kd7-ixt1.vercel.app/create/X27AKmYCEzgl?dataSource=Prod)
2. Add a second Chart 
3. Click on any Chart
4. Notice how Add Label Button opens Add Label Options
5. Scroll down to "Title & Description" Section Notice there is no way to edit tab label

**How to test?**
1. Go to this [Link](https://visualization-tool-fdkin1fbd-ixt1.vercel.app/create/uM3BIxyxSCxv?dataSource=Prod)
2. Add a second Chart
3. Click on any Chart
4. Notice how Chart Changes without opening Tab Label Editor 
5. Click Dropdown Arrow
6. See new Edit tab Label - Click on it and edit label
7. Scroll down to "Title & Description" Section Notice you can edit tab title here as well.

### CODE REVIEW

File: app/locales/de/messages.po [hunk 1]
```diff
@@ -13,1567 +13,1115 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: app/configurator/components/chart-configurator.tsx
 msgid "Add filter"
 msgstr "Filter hinzufügen"
```
Comment (bprusinowski) [inline app/locales/de/messages.po:18]: It looks like we lost add string usage location comments? 😱 It's sometimes helpful to see where the string is being used directly in the `po` files, I think you can add them back via `locales:extract` 👍

**Review by bprusinowski — APPROVED** [review]
LGTM, thanks!

File: app/locales/de/messages.po (hunk 1 above)
Comment (noahonyejese) [reply app/locales/de/messages.po:18]: my bad!

Comment (bprusinowski) [reply app/locales/de/messages.po]: Thanks!

**Pull request merged**